src/azul/terraform.py

```diff
@@ -815,24 +815,43 @@ def tf_config(self, app_name):
         key = 'x-amazon-apigateway-minimum-compression-size'
         openapi_spec[key] = config.minimum_compression_size
         assert 'aws_api_gateway_gateway_response' not in resources, resources
-        openapi_spec['x-amazon-apigateway-gateway-responses'] = {
-            f'DEFAULT_{response_type}': {
-                'responseParameters': {
-                    # Static value response header parameters must be enclosed
-                    # within a pair of single quotes.
-                    #
-                    # https://docs.aws.amazon.com/apigateway/latest/developerguide/request-response-data-mappings.html#mapping-response-parameters
-                    # https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions-gateway-responses.html
-                    #
-                    # Note that azul.strings.single_quote() is not used here
-                    # since API Gateway allows internal single quotes in the
-                    # value, which that function would prohibit.
-                    #
-                    f'gatewayresponse.header.{k}': f"'{v}'"
-                    for k, v in AzulChaliceApp.security_headers.items()
-                }
-            } for response_type in ['4XX', '5XX']
-        }
+        openapi_spec['x-amazon-apigateway-gateway-responses'] = (
+            {
+                f'DEFAULT_{response_type}': {
+                    'responseParameters': {
+                        # Static value response header parameters must be enclosed
```

Comment:
This comment should be moved up so it is evident that it applies to both dictionaries.